README.md

```diff
@@ -2,11 +2,17 @@
 
 [![Build Status](https://travis-ci.org/jts/nanopolish.svg?branch=master)](https://travis-ci.org/jts/nanopolish)
 
-Software package for signal-level analysis of Oxford Nanopore sequencing data.
+Software package for signal-level analysis of Oxford Nanopore sequencing data. Nanopolish is designed to calculate a new consensus sequence for a draft genome assembly produced by a program like [CANU](https://github.com/marbl/canu). It does not error correct reads.
```

Comment:
Nanopolish does more than calculate a consensus sequence, that is just one of its features. It will also call base modifications (`nanopolish call-methylation`), align events to a reference genome (`nanopolish eventalign`) and call SNPs and indels with respect to a reference genome (`nanopolish variants`). See the nanopolish modules section